modules/ndu/src/backend/ndu-engine.ts

```diff
@@ -283,18 +283,30 @@ export class UnderstandingEngine {
     const electedTrigger = event.ndu.triggers[actionToTrigger[topAction]]
 
     if (electedTrigger) {
-      switch (electedTrigger.trigger.type) {
+      const { trigger } = electedTrigger
+
+      switch (trigger.type) {
         case 'workflow':
-          event.ndu.actions = [
-            {
+          const sameWorkflow = trigger.workflowId === currentFlow?.replace('.flow.json', '')
+          const sameNode = trigger.nodeId === currentNode
+
+          event.ndu.actions = [{ action: 'continue' }]
+
+          if (sameWorkflow && !sameNode) {
```

Comment:
I thought this action existed as "continue workflow execution"

Reply:
Continue workflow is normally when you flow from one node to another. Here you might be on a node, but a trigger makes you jump to another one (with higher confidence), so it's not really a "continue". 

Right now, a jump will say "Starting workflow", which is definitely not the case

Reply:
Oh right I understand makes you potentially skip nodes and flow you to a different trigger within the same workflow ... then "start workflow" still makes sens because we restart the workflow from somewhere else.

Is there a use case where you could jump from a node to another within the same workflow without restarting the flow ?

Reply:
I don't think we should consider it "start workflow" in that case, because let's say the workflow is "approval", then the workflow goal is to approve something. There may be different questions or "jumps" to complete this, but in the end, you completed the workflow and approved something.

And it would be a bit misleading in analytics, because "workflow started" happens when the workflow name changes to another one, not when you jump on another node